source/includes/opts/acceptDisclaimer.rst

```diff
@@ -0,0 +1,14 @@
+
+Accepts disclaimers for the :ref:`embedded verifier
+<c2c-verifier-disclaimer>` and :ref:`Beta <c2c-beta-program>`
+features (when using {+c2c-beta-program-short+}).
+
+When the ``mongosync`` process starts without this option, the
+user is prompted to accept each disclaimer.
+
+This option allows users who have already read and accepted
+disclaimers to skip the prompt and start the ``mongosync``
+process with the features enabled.
```

Comment:
I don't know if we should say `with the features enabled`. For example, for a sharded migration, even if you accept the disclaimer we won't actually enable the verifier since it isn't compatible.

Reply:
Fixed